test/src/tests/runner.go

```diff
@@ -10,37 +10,44 @@ import (
 	"tests/generated/report"
 )
 
+const testServerPath = "../../../node_modules/@microsoft.azure/autorest.testserver"
+
 func main() {
-	server, err := startServer()
+	err := startServer()
 	if err != nil {
-		fmt.Printf("Error! %v\n", err)
+		fmt.Printf("Error starting server: %v\n", err)
 	}
 	allPass := true
 	runTests(&allPass)
 	getReport()
 	getAzureReport()
-	server.Kill()
+	err = stopServer()
+	if err != nil {
+		fmt.Printf("Error stopping server: %v\n", err)
+	}
 	if !allPass {
 		fmt.Println("Not all tests passed")
 		os.Exit(1)
 	}
 }
 
-func startServer() (*os.Process, error) {
+func startServer() error {
 	fmt.Println("Go Tests.......")
-	testServerPath := "../../../node_modules/@microsoft.azure/autorest.testserver"
 	install := exec.Command("npm", "install")
 	install.Dir = testServerPath
 	server := exec.Command("npm", "start")
 	server.Dir = testServerPath
 
 	if err := install.Run(); err != nil {
-		return install.Process, err
-	}
-	if err := server.Start(); err != nil {
-		return server.Process, err
+		return err
 	}
-	return server.Process, nil
+	return server.Start()
```

Comment:
Same as the comment below, I think we should wait to return from this method until the server has finished starting. (Unless I'm wrong about how this process lives.)

Reply:
Running "npm start" doesn't return so we have to call Start() in this case.